Services/LTI/classes/class.ilLTICronOutcomeService.php

```diff
@@ -31,6 +31,13 @@ public function getId()
 		return 'lti_outcome';
 	}
 
+	// cat-tms-patch start
+	public function getTitle()
+	{
+		return "lti outcome cron job.";
```

Comment:
Sollte das nicht gegen ILIAS-Core gestellt sein? dann kann man sich // cat-tms-patch start/end sparen

Reply:
Wenn wir das an den Core stellen, dann ist das ein Improvement und geht nur in den Trunk. Da wir auf 5.3 arbeiten brauchen wir den Patch trotzdem.